packages/oc-pages/dashboard/dashboard.vue

```diff
@@ -44,10 +45,9 @@ export default {
     async mounted() {
         const pathComponents = this.$router.options.base.split('/');
         while(pathComponents.length > 0 && pathComponents[0] == '') pathComponents.shift();
-        const currentNamespace = pathComponents.includes('dashboard')? 
-            pathComponents.slice(0, Math.max(0, pathComponents.lastIndexOf('dashboard'))).join('/'):
-            pathComponents.slice(1).join('/');
+        const currentNamespace = pathComponents.slice(0, -1).join('/')
```

Comment:
If all this `while` is doing is stripping empty elements from leading/trailing slashes in the route, this can be a simple `filter()`, as empty strings are falsey:
```js
const pathComponents = this.$router.options.base.split('/').filter(p => p);
const currentNamespace = pathComponents.join('/');
```